scripts/travis.sh

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env bash
```

Comment:
Possibly add an option to apollo.sh to skip sourcing it, otherwise keeping this and apollo.sh in sync might be a pain.